client/src/connection/rest/index.ts

```diff
@@ -146,6 +157,28 @@ async function setup(): Promise<void> {
   setContextValue("SAS.sessionId", computeSession.sessionId);
 }
 
+/**
+ * Formats the connection profile sasOptions into a format that the compute
+ * API can understand.
+ *
+ * Examples:
+ *
+ * ```
+ * ["-PAGESIZE=MAX"] -> ["-PAGESIZE MAX"]
+ * ["-NOTERMINAL"] -> ["-NOTERMINAL"]
+ * ```
+ *
+ * @returns formatted SAS Options
+ */
+function formatSASOptions() {
+  const formattedOpts = config.sasOptions.map((opt) => {
+    let formatted = opt;
+    formatted = formatted.replace(/-/gi, "");
```

Comment:
Hi @smorrisj, this looks confusing to me. Shouldn't it be
```
formatted = formatted.replace(/=/gi, " ");
```

Reply:
> Hi @smorrisj, this looks confusing to me. Shouldn't it be
> 
> ```
> formatted = formatted.replace(/=/gi, " ");
> ```

We want to turn:
-PAGESIZE max
into:
PAGESIZE max

Based on this, its appropriate to replace the dash with empty?

Reply:
> Hi @smorrisj, this looks confusing to me. Shouldn't it be
> 
> ```
> formatted = formatted.replace(/=/gi, " ");
> ```

This sounds correct. @Becky-Williamson is it possible to log a bug for this? We'll need to fix it on a separate PR since this has been closed.